### ISSUE TITLE
Add each algo difficulty to api response

### ISSUE THREAD

**Issue by Fredar** [mimblewimble/grin#2688]
Right now we only have the base diff (target_difficulty) in the api response. Example:

https://api.grinmint.com/v1/networkStats
{
  "status": true,
  "height": 88987,
  "last_block_pushed": "00000544319fe2f6846dfbe97440c9997917dc65da460be3cd82bd484eba5b10",
  "prev_block_to_last": "000000646cf9c8efd2b2bf0cd47471748b7c0ec59e67b761f4525f18247fd90b",
  "target_difficulty": 3917679356,
  "total_difficulty": 259489271059764,
  "primary_hashrate": 65656.67,
  "secondary_hashrate": 1577645.02
}

In the current situation its still possible to calculate proper cr29 diff via multiplier and ar_scale param.
We can also calculate ct31 diff, but only if its the only primary algo. 
As soon as you add a second primary algo then its impossible to calculate difficulties for them cause the block distribution per algo is not known. (At least not on the api level)

Please consider adding each algo difficulties to networkStats response before you add more primary algos.

**Comment by jaspervdm** [comment]
> ..before you add more primary algos
What do you mean by this exactly? Are you refering to the bigger cuckatoo sizes? They are already accepted by the network, it's just so far there isn't a (public) miner that is capable of mining them.

**Comment by Fredar** [comment]
@jaspervdm Ok was not aware of that.
Would rephrase it then to `before miners are able to solve bigger cuckatoo algos`

Bottom line is that it would be impossible for calculators to provide estimation without each algo diff.
At least for those that rely on algo difficulty instead of nethash approx.

**Comment by quentinlesceller** [comment]
See https://api.grinmint.com/v2/networkStats. Closing.
